libs/ui/src/tokens/components/atoms/_icon.css

```
@@ -101,6 +141,42 @@
+  @apply icon-[mdi--alert];
+}
+
+@utility token-icon-alert-circle {
+  @apply icon-[mdi--alert-circle];
+}
+
+@utility token-icon-information {
+  @apply icon-[mdi--information];
+}
+
+@utility token-icon-ellipsis-horizontal {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)